bin/otioview.py

```diff
@@ -181,8 +183,14 @@ def main():
     argument_map = {}
     for pair in args.adapter_arg:
         if '=' in pair:
-            key, val = pair.split('=')
-            argument_map[key] = val
+            key, val = pair.split('=', 1)  # only split on the 1st '='
+            try:
+                # Sometimes we need to pass a bool, int, list, etc.
+                parsed_value = ast.literal_eval(val)
+            except (ValueError, SyntaxError):
```

Comment:
Wondering if these should fall back on strings - it could hide unexpected behavior from people, but I don't feel strongly about it.

Reply:
Without the fallback, you would have to put `-a key=\"value\"` on the command line to get a python literal string.